x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlResponseListener.java

```diff
@@ -31,72 +26,36 @@
  * Listens for a single {@link EsqlQueryResponse}, builds a corresponding {@link RestResponse} and sends it.
  */
 public class EsqlResponseListener extends RestResponseListener<EsqlQueryResponse> {
```

Comment:
This class has no intrinsic value and its content moved over to the RestEsqlQueryAction.

Reply:
To clarify: do you mean `EsqlResponseListener`? If so, I agree that it could probably also be a nested (private static) class inside `RestEsqlQueryAction`; however, this already existed before any recent logging PRs so refactoring this would be out of scope for this PR IMO.